cita-executor/core/src/cita_executive.rs

```diff
@@ -113,3 +357,34 @@ impl From<AuthenticationError> for ExecutionError {
         }
     }
 }
+
+/// Transaction execute result.
+#[derive(Debug)]
+pub struct TxExecResult {
+    /// Final amount of gas left.
+    pub gas_left: U256,
+}
+
+#[derive(Clone, Debug)]
+pub struct ExecParams {
+    /// Address of currently executed code.
+    pub code_address: Address,
+    /// Sender of current part of the transaction.
+    pub sender: Address,
+    /// Gas paid up front for transaction execution
+    pub gas: U256,
+    /// Input data.
+    pub data: Option<Bytes>,
+}
+
+impl Default for ExecParams {
+    /// Returns default ActionParams initialized with zeros
+    fn default() -> ExecParams {
+        ExecParams {
+            code_address: Address::new(),
+            sender: Address::new(),
+            gas: U256::zero(),
+            data: None,
+        }
+    }
+}
```

Comment:
Could we put struct `ExecParams`, `TxExecResult` out in a separated `.rs`, and make cita_executive more clearly and simple ?  FYI

Reply:
I will refine it later.